### DISCUSSION TITLE
Color request for WhatsApp

### DISCUSSION THREAD

**Discussion by DarkD** [catppuccin/catppuccin#2265] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Hello, I'd like to request green-ish variation of your theme for WhatsApp. Previously I've made one myself, but with update lost it and now remade again. I wanted something with similiar contrast and brightness as Mocha variation, but green, to match other themes I'm using for browser, etc..
That's what I'm using now:
`@emerald:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #b5e9e5; @subtext1: #aecccc; @subtext0: #96a6a6; @overlay2: #87a69a; @overlay1: #739186; @overlay0: #617b72; @surface2: #4b6e60; @surface1: #3b5951; @surface0: #29463b; @base: #152a29; @mantle: #0e1d1c; @crust: #081312; };`
If it's possible include as is or edit it and include for use by me and others. I just don't want to lose it with next update. If it's not possible then I'd ressort to storing it somewhere and restore on updates and I'm sorry that I've bothered you. Thanks,

### Submission Guidelines

- [ ] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:, we wouldn't really accept this into the mainline theme as it's a separate flavor from flavors that make up Catppuccin right now. 

I'd encourage you to [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo#forking-a-repository) the [catppuccin/userstyles](https://github.com/catppuccin/userstyles) repository and add in your extra flavour there. Then you can automatically pull in updates to the theme and optionally resolve any conflicts yourself in git if there are any.

Hope that's okay!
